data/anthony2021early/anthony2021early.yaml

```diff
@@ -0,0 +1,317 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Early induction of functional SARS-CoV-2-specific T cells associates with rapid
+  viral clearance and mild disease in COVID-19 patients
+doi: 10.1016/j.celrep.2021.108728
+description: >-
+  Measured SARS-CoV-2 viral load in the upper respiratory tract, along with SARS-CoV-2-specific
+  antibodies and T cell responses, at multiple time points from acute infection through
+  convalescence or until death.
+analytes:
+  swab_SARSCoV2_N:
+    description: >-
+      The swab_SARSCoV2_N metric refers to the detection of SARS-CoV-2 RNA in patient
+      swabs,using RT-PCR cycle counts as a proxy for viral quantity. Data for the
+      nasopharyngeal swab results were obtained from the combined dataset in the supplementary
+      materials of Challenger et al. (2022)
+    limit_of_quantification: unknown
+    limit_of_detection: unknown
```

Comment:
Figure 1 has some information about the limit of detection (dashlines at -11?). Need to check what this means. Also, there are negative samples (usually last two observations) for each patient but currently not included in the Challenger data. We also need to include them.

Reply:
the dotted line represents a quantity of 10^-12,  corresponding to the lower detection limit.(below this line are considered undetectable).
I checked the Challenger data (study16) again. It seems like everyone included last two negative observations. Could you give me more details about it?

Reply:
I read the paper again, and it seems like 2^(-CT)=10^(-11), so the limit of detection should be approximately 36.5. Actually I am still not sure about the answer.

Reply:
That's right. You can use CT value 36.5 as the limitation of detection. Thank you for checking on this. The unit will be `circle threshold`.

Reply:
Regarding the units, I've typically seen `cycle threshold` rather than `circle threshold` for the qPCR data.

Reply:
Sorry, that's a typo. I am not sharp enough with current illness.